src/Campaigns/ValueObjects/CampaignType.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Give\Campaigns\ValueObjects;
+
+use Give\Framework\Support\ValueObjects\Enum;
+
+/**
+ * @unreleased
+ *
+ * @method static CampaignStatus CORE()
+ * @method static CampaignStatus P2P()
+ * @method bool isCore()
+ * @method bool isP2p()
+ */
+class CampaignType extends Enum
+{
+    const CORE = 'core';
+    const P2P = 'p2p';
```

Comment:
Should we start referring to P2P using the pull name, like `const PEER_TO_PEER` and `isPeerToPeer()`? Now that we are moving into a larger context, I don't want shorthand and abbreviation to create ambiguity.

Reply:
Good point! Resolved https://github.com/impress-org/givewp/pull/7520/commits/a72f80c29861921f5c493dfbf22a550abf3e7e70